admin/server/middleware/browserify.js

```diff
@@ -100,12 +117,18 @@ module.exports = function (file, name) {
 		});
 	}
 	function serve (req, res) {
-		if (!ready) {
-			build();
-			queue.push([req, res]);
-			return;
-		}
-		send(req, res);
+		fs.readFile(outputPath, (err, data) => {
```

Comment:
can use this regardless of the user enabling the new "write to file" workflow since if they haven't the file shouldn't exist. not optimal performance (one file read that will fail with another predicate check) but hardly going to be the largest issue in this workflow.